AssemblyCSharp/Mod/Auto/AutoChat/AutoChat.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Mod.ModHelper;
+using Mod.ModHelper.CommandMod.Chat;
+using Mod.ModHelper.Menu;
+using System.IO;
+namespace Mod.Auto.AutoChat
+{
+    public class AutoChat : ThreadActionUpdate<AutoChat>
+    {
+        //Có thể viết tự điều chỉnh time auto mà lười quá
```

Comment:
Các phần có thể bổ sung sau nên sử dụng `//todo:` để sau có thể tìm lại nhé
```suggestion
        //todo: Có thể viết tự điều chỉnh time auto mà lười quá
```